.github/workflows/ruby_tests.yml

```diff
@@ -4,94 +4,26 @@ on:
   push:
     branches:
     - master
+
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
 env:
   RAILS_ENV: test
   DATABASE_URL: postgresql://postgres:@localhost/test
   DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+
 jobs:
   rubocop:
-    runs-on: ubuntu-latest
-    env:
-      BUNDLE_FOREMAN: '0'
-      BUNDLE_WITHOUT: development:test
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: 2.7
-          bundler-cache: true
-      - name: Run rubocop
-        run: bundle exec rubocop -P --format github
-  test_ruby:
-    runs-on: ubuntu-latest
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+    with:
+      command: bundle exec rubocop -P --format github
+      
+  test:
+    name: Ruby
     needs: rubocop
-    env:
-      BUNDLE_WITHOUT: journald:development:console:libvirt
-    services:
-      postgres:
-        image: postgres:12.1
-        ports: ['5432:5432']
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
-    strategy:
-      fail-fast: true
-      matrix:
-        foreman-repo: [theforeman/foreman]
-        foreman-core-branch: [develop]
-        ruby-version: [2.7]
-        node-version: [12]
-    steps:
-      - run: |
-          sudo apt-get update
-          sudo apt-get install build-essential libcurl4-openssl-dev zlib1g-dev libpq-dev
-      - name: Install chromedriver
-        uses: nanasess/setup-chromedriver@master
```

Comment:
@ekohl i can't see this step execution in the reusable action, do we want to execute this one? i'm not aware of this one.

Reply:
We need a browser for the acceptance tests. I think a separate container that runs selenium together with https://github.com/theforeman/foreman/pull/9952 could be an alternative. It would be best to track this by opening an issue on https://github.com/theforeman/actions or at least mention it in https://github.com/theforeman/actions/issues/1

Reply:
https://github.com/theforeman/actions/issues/38